mix.exs

```diff
@@ -36,7 +36,8 @@ defmodule Guardian.DB.Mixfile do
   defp deps do
     [
       {:guardian, "~> 1.0"},
-      {:ecto, "~> 2.2"},
+      {:ecto, "~> 3.0"},
+      {:ecto_sql, "~> 3.0.0-rc.1"},
```

Comment:
Hey @tanweerdev is there an official release of `ecto_sql`?

Reply:
It's release was delayed, but ecto_sql 3.0.0 final is available today.

Reply:
Thanks @nathany (sorry I was away)

Reply:
np. thank you!

Reply:
@nathany I have looked at code and its good.

Reply:
@tanweerdev thanks a lot for the contribution, really appreciated.

Since @nathany version has the official released of `ecto_sql` and he was working on this before I will take his PR.